crates/rolldown/tests/esbuild/dce/inline_identity_function_calls/artifacts.snap

```diff
@@ -1,6 +1,38 @@
 ---
 source: crates/rolldown_testing/src/integration_test.rs
 ---
+# warnings
+
+## PARSE_ERROR
+
+```text
+[PARSE_ERROR] Warning: Identifier `DROP` has already been declared
+   ╭─[identity-last.js:1:10]
+   │
+ 1 │ function DROP(x) { return [x] }
```

Comment:
Why do I feel like all these are script mode code.

Reply:
https://github.com/rolldown/rolldown/blob/e47e8611a818d28c300160d811907c2306bcb5a4/crates/rolldown/src/utils/parse_to_ecma_ast.rs#L92-L100

https://github.com/rolldown/rolldown/blob/e47e8611a818d28c300160d811907c2306bcb5a4/crates/rolldown/src/utils/parse_to_ecma_ast.rs#L20-L28

Base on the code, they are parsed as module.